### PR TITLE
SFR-1383 paginate by default

### PR DESCRIPTION
**Pull request by hokei**
Starts the web reader in pagination mode 
Exposes settings so that a user can control which mode the reader starts in

### CODE REVIEW

File: src/PdfReader/index.tsx [hunk 1]
```diff
@@ -377,8 +400,10 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
       navigator: {
         goForward,
         goBackward,
-        zoomIn,
-        zoomOut,
+        increaseFontSize,
+        decreaseFontSize,
+        setFontFamily,
+        setColorMode,
```
Comment (samanthaandrews) [inline src/PdfReader/index.tsx]: Should these be in the PdfReader?

File: src/ui/Header.tsx [hunk 2]
```diff
@@ -89,6 +89,7 @@ export const HeaderWrapper: React.FC<ComponentProps<typeof Flex>> = ({
       px={8}
       borderBottom="1px solid"
       borderColor="gray.100"
+      backgroundColor="ui.white"
```
Comment (samanthaandrews) [inline src/ui/Header.tsx]: i noticed that in scroll mode you could see the text behind the header. making header background white fixes it.

Comment (Toxiapo) [reply src/ui/Header.tsx]: We do want to support color mode here, so you might want to pass down the `bgColor` prop from the header

File: src/HtmlReader/index.tsx [hunk 3]
```diff
@@ -137,7 +149,13 @@ export default function useHtmlReader(args: ReaderArguments): ReaderReturn {
     }).then((reader) => {
       dispatch({ type: 'SET_READER', reader });
     });
-  }, [webpubManifestUrl, getContent, injectables, injectablesFixed]);
+  }, [
+    webpubManifestUrl,
+    getContent,
+    injectables,
+    injectablesFixed,
+    readerSettings,
```
Comment (Toxiapo) [inline src/HtmlReader/index.tsx]: stray `readerSettings`

File: src/ui/Header.tsx (hunk 2 above)
Comment (samanthaandrews) [reply src/ui/Header.tsx]: good catch! fixed now

File: src/HtmlReader/index.tsx (hunk 3 above)
Comment (samanthaandrews) [reply src/HtmlReader/index.tsx]: i think this is meant to be here, but i'll let @hokei confirm

Comment (hokei) [reply src/HtmlReader/index.tsx]: I think this can be taken out, as it is not used.

**Review by samanthaandrews — APPROVED** [review]

**Pull request merged**